spec/factories.rb

```diff
@@ -1,14 +1,19 @@
 FactoryBot.define do
   factory :family do
-    last_name "Liso"
-    capability 3
+    last_name { FFaker::NameIT.last_name }
+    first_name { FFaker::NameIT.first_name }
+    address { ["San Cesareo", "Zagarolo", "Colonna"].sample }
+    capability { [2, 3, 4, 5].sample }
+    animals { ["2 dogs", "5 cats", "1 horse", "ducks"].sample }
+    preferred_gender { ["Female", "Male"].sample }
   end
 
   factory :group do
-     name "St. Luis"
-     responsable_name "Miss Faget"
-     responsable_contact "0123456789"
-     start_date 1.month.from_now
-     end_date 2.months.from_now
+    name { FFaker::AddressFR.city }
+    responsable_name { FFaker::NameFR.name }
+    responsable_contact { FFaker::PhoneNumberFR.mobile_phone_number }
+    start_date 1.month.from_now
+    end_date 2.month.from_now
+    notes { FFaker::Lorem.phrase }
```

Comment:
Since notes is not required, I would not add them to the factory

Reply:
Ok, I added it just to see the effect of a full table! I suppose now I can delete it..

Reply:
Again, we can work on cleaning this up tomorrow